<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pandapower</class>
 <widget class="QTabWidget" name="pandapower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pandapower gui</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="main">
   <attribute name="title">
    <string>Main</string>
   </attribute>
   <widget class="QPushButton" name="main_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Network</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="main_message">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>501</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="main_solve">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Solve (runpp)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_empty">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty Network</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_basic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Basic Results</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="inspect">
   <attribute name="title">
    <string>Inspect</string>
   </attribute>
   <widget class="QTextBrowser" name="inspect_message">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>501</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_lines">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_bus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_switch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_sgen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stat Gen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_ext_grid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>External Grid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_trafo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_gen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_xward">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Extended Ward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_impedance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Impedance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_dcline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DC Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_measurement">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_shunt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shunt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_ward">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inspect_trafo3w">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3W-Transformer</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="html">
   <attribute name="title">
    <string>HTML</string>
   </attribute>
   <widget class="QWebView" name="html_webview">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>611</width>
      <height>491</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
    <property name="renderHints">
     <set>QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="html_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HTML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="html_show_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HTML</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="results">
   <attribute name="title">
    <string>Results</string>
   </attribute>
   <widget class="QPushButton" name="res_bus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_ward">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ward</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="res_message">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>621</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="res_trafo3w">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3W-Transformer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_ext_grid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>External Grid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_measurement">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_trafo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_shunt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shunt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_gen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_xward">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Extended Ward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_lines">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_impedance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Impedance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_dcline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DC Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_switch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res_sgen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stat Gen</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="build">
   <attribute name="title">
    <string>Build</string>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
